### ISSUE TITLE
SVG export error

### ISSUE THREAD

**Issue by zsviczian** [excalidraw/excalidraw#3602]
[example.zip](https://github.com/excalidraw/excalidraw/files/6510625/example.zip)
![image](https://user-images.githubusercontent.com/14358394/118856447-3fa21b80-b8d7-11eb-81bb-4f761d6e358f.png)
![image](https://user-images.githubusercontent.com/14358394/118856221-0073ca80-b8d7-11eb-95bd-028ec8dd5b81.png)

**Comment by ad1992** [comment]
Yeah I am able to reproduce, its only happening for svg.

**Comment by pshihn** [comment]
The shapes that are not rendered by SVG, the color is set to `71ba0` which is an invalid color for svg.  The app probably parses the color and falls back to black.

**Comment by ad1992** [comment]
Yep, that's right, thanks for pointing this out @pshihn. 
@zsviczian can you fix the value for stroke color? & it should work.
We should probably show some error message to inform the users about this.

**Comment by zsviczian** [comment]
Thanks. I did not spot the issue with the color. will fix!

**Comment by dwelle** [comment]
As I understand that color was set by an script that automates the diagram in your Obsidian plugin? The problem then is we're not restoring colors and allow importing invalid ones. It's unclear at what point does the renderer fall back to `#000` and whether we can detect it and update the element's color.

**Comment by zsviczian** [comment]
Correct. I have a silly demo script to generate a mindmap from text. To make it more colorful I added the following to change color at every first level branch...
`if (depth == 1) ea.style.strokeColor = '#'+(Math.random()*0xFFFFFF<<0).toString(16);`
I am guessing this resulted in the invalid color.
You may want to add validation of imported colors, though I wouldn't worry about it too much. This is not an excalidraw issue, but an issue in my automation script.

Thanks.

**Comment by pshihn** [comment]
@zsviczian i think your random calculator does not take into account leading 0s. So it could produce `6dbcc` when it probably meant `06dbcc`
